frontend/src/admin/components/dropdown-menu/dropdown-menu.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../../styles/main";
+
+.c-dropdown-menu {
+  position: relative;
+
+  button {
+    border: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .c-dropdown-menu-items {
+    display: none
+  }
+
+  &--active {
+    position: relative;
+
+    button {
+      border: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    .c-dropdown-menu-items {
```

Comment:
This is a separate component than the `.c-dropdown-menu`, and should have its own file. By doing this you also encourage the independence of the two components.